File: 242-ValidAnagram/242-ValidAnagram.java
// Last updated: 7/13/2026, 10:10:39 AM
class Solution {
    public boolean isAnagram(String s, String t) {
        if(s.length()!=t.length()) return false;
        char c[]=s.toCharArray();
        char c1[]=t.toCharArray();
        Arrays.sort(c);
        Arrays.sort(c1);
        return new String(c).equals(new String(c1));


       
    }
}

File: 242-ValidAnagram/242-ValidAnagram.py
# Last updated: 7/13/2026, 10:10:52 AM
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        # if len(s) != len(t):
        #     return False
        # x = set(s)
        # for i in x:
        #     if s.count(i) != t.count(i):
        #         return False
        # return True
        return Counter(s) == Counter(t)
        